spec/support/payloads/document.rb

```diff
@@ -303,7 +303,7 @@ def self.drug_safety_update_content_item(attrs = {})
     }.deep_merge(attrs)
   end
 
-  def self.employment_tribunal_decision_content_item(attrs)
+  def self.employment_tribunal_decision_content_item(attrs = {})
```

Comment:
Yes, but could we defer work on the factories to another PR? I did the minimum to keep things working here but would like to revisit the test factories wholesale on another PR.